sdk/servicebus/azure-servicebus/api_review.py

```diff
@@ -210,6 +217,69 @@ def get_token(self, *scopes, **kwargs):
 
 class AutoLockRenew:
     def __init__(self, executor=None, max_workers=None):
-    def __enter__(self):
-    def __exit__(self, *args):
-    def register(self, renewable, timeout=300):
\ No newline at end of file
+        def __enter__(self):
+
+            def __exit__(self, *args):
+
+            def register(self, renewable, timeout=300):
```

Comment:
Do you have a rule to suffix timeout with some sort of measurement? Ie, this must be seconds so `timeoutSeconds` or something of that nature?

Reply:
Also - what does `renewable` do here? Isn't it renewable by default? Does this enable something interesting?